src/content/utils/isFeaturesObject.ts

```diff
@@ -0,0 +1,13 @@
+import { hasOwnProperty } from "ts-type-safe";
+import { Features } from "../../services";
+
+export const isFeaturesObject = (features: unknown): features is Features => {
```

Comment:
Das macht `zod` für uns out-of-the-box. Schau Mal, wie wir's beim `settingsSchema` machen:

![image](https://github.com/user-attachments/assets/d97738be-8814-4677-87f1-c8f3b2fe19b6)